app/src/main/java/org/mozilla/focus/FocusApplication.kt

```diff
@@ -27,6 +28,8 @@ class FocusApplication : LocaleAwareApplication() {
         super.onCreate()
 
         PreferenceManager.setDefaultValues(this, R.xml.settings, false)
+        WebViewProvider.readEnginePref(this)
+        WebViewProvider.writeEnginePref(this)
```

Comment:
I wonder why we immediately write here? Don't we only need to do that if the value was changed, either by some internal QA thingy or from the A/B config later?

Reply:
I was persisting the current state of the engine on startup to verify that the engine was sticky if I set the preference. The read is required. I can remove the write in this case.